src/sentry/static/sentry/app/views/projectDetail/projectLatestAlerts.tsx

```diff
@@ -38,11 +39,25 @@ type State = {
 } & AsyncComponent['state'];
 
 class ProjectLatestAlerts extends AsyncComponent<Props, State> {
+  shouldComponentUpdate(nextProps: Props, nextState: State) {
+    if (
+      !isEqual(this.state, nextState) ||
```

Comment:
Do you need a deep compare of state?

Reply:
Yeah, I probably don't.
It's changed now 👍